notebooks/QuickStart.ipynb

```diff
@@ -76,7 +76,7 @@
    "source": [
     "### c) Define server configuration\n",
     "\n",
-    "The easiest way to set up a configuration is to paste a JSON configuration string into `adb config create`.\n",
+    "The easiest way to set up a configuration is to paste an ApertureDB key string into `adb config create`.\n",
```

Comment:
```suggestion
    "The easiest way to set up a configuration is to paste an ApertureDB Key string into `adb config create`.\n",
```

Reply:
do you think we should standardize on "ApertureDB Key" or "ApertureDB key" ? I think I have it as 'key' elsewhere.

Reply:
We should standardise.  I thought I saw "ApertureDB Key" elsewhere.